### DISCUSSION TITLE
Are you considering adding the carousel table feature?

### DISCUSSION THREAD

**Discussion by llds66** [graphieros/vue-data-ui#68] [Q&A] [answered]
Similar to DataV carousel component
[DataV scrollBoard](http://datav.jiaminghi.com/guide/scrollBoard.html#%E5%9F%BA%E6%9C%AC%E8%BD%AE%E6%92%AD%E8%A1%A8)

**Comment by graphieros** [comment]
Hi @llds66
That's a very nice feature. I will look into it:)

Thanks

**Reply by llds66** [reply]
I'm looking forward to it

**Comment by graphieros** [comment, marked as answer]
Hi @llds66 

New component added in [v2.3.6](https://github.com/graphieros/vue-data-ui/releases/tag/v2.3.6)

**Reply by llds66** [reply]
Thanks for your work, I'm using it at work, very timely.

**Reply by graphieros** [reply]
You are welcome:)
Use v2.3.7 which adds touch events, to pause animation when user scrolls on touch screens
